Admin/ORM/MediaAdmin.php

```diff
@@ -36,7 +36,7 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
             ->add('context', null, array(
                 'show_filter' => $this->getPersistentParameter('hide_context') !== true,
             ), 'choice', $options)
-            ->add('category', null, array(
+            ->add('category', 'doctrine_orm_choice', array(
```

Comment:
I dont know, I am not dedicated Sonata user, just started to learning damn thing. I don't know how it did worked before, now it does not :)

If you point me what to checkout and debug, I can take a look and try to fix it. Or you can do it, I don't mind -> issue exists, I think that you can reproduce it.